proxy_agent/src/proxy/proxy_authentication.rs

```diff
@@ -109,7 +109,7 @@ pub trait Authenticate {
     fn authenticate(
         &self,
         connection_id: u128,
-        request_url: String,
+        request_url: hyper::Uri,
```

Comment:
nit: update comments for the code example on top of this file

Reply:
thanks, updated.